pages/about/team.js

```diff
@@ -0,0 +1,87 @@
+// Module imports
+import marked from 'marked'
+import React from 'react'
+
+
+
+
+
+// Component imports
+import wordpressService from '../../services/wordpress'
+import Component from '../../components/Component'
+import Page from '../../components/Page'
+
+
+
+
+
+// Component constants
+const title = 'Team'
+
+
+
+
+
+class Team extends Component {
+  /***************************************************************************\
+    Public Methods
+  \***************************************************************************/
+
+  static async getInitialProps (/*{ query, store }*/) {
+    const response = await wordpressService.get('/wp-json/wp/v2/users')
+
+    return { team: response.data }
+  }
+
+  render () {
+    const { team } = this.props
+
+    return (
+      <React.Fragment>
+        <header>
+          <h1>Meet the Team</h1>
+        </header>
+
+        <ul className="card-list">
+          {team.map(teamMember => {
+            const {
+              acf,
+              description,
+              id,
+              name,
+            } = teamMember
+
+            return (
+              <li className="card" key={id}>
+                <header>
+                  <div
+                    aria-label={`${name}'s avatar`}
+                    className="avatar small"
+                    style={{ backgroundImage: `url(${teamMember.avatar_urls['96']})` }} />
+
+                  <h2>{name}</h2>
+
+                  <aside>
+                    <small>{acf.rfg_title}</small>
+                  </aside>
+                </header>
+
+                {/* eslint-disable react/no-danger */}
+                <div
+                  className="content"
+                  dangerouslySetInnerHTML={{ __html: marked(description) }} />
```

Comment:
I would rather us use a react library like [`react-markdown`](https://rexxars.github.io/react-markdown/) to achieve this.

Reply:
Under further inspection, we should be fine here. Though, It would be nice if we could get a dedicated component to handle this to better denote we're expecting markdown in the input. Just to keep things readable..